.github/workflows/nightlyAndMainImage.yml

```
@@ -50,11 +47,17 @@ jobs:
     runs-on: ubuntu-latest
     env:
       DOCKER_IMAGE_NAME: newrelic/nri-kubernetes
-      DOCKER_IMAGE_TAG: nightly
       DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm" # Must be consistent with the matrix from the job above
     steps:
       - uses: actions/checkout@v2
-
+      - name: Setting Docker image tag
+        run: |
+          if [[ "${{ github.ref_name }}" == "main" ]]
+          then
+          echo "DOCKER_IMAGE_TAG=main" >> $GITHUB_ENV
+          else
+          echo "DOCKER_IMAGE_TAG=nightly" >> $GITHUB_ENV
+          fi
```

I would use a more compact syntax and indentation

```suggestion
          if [[ "${{ github.ref_name }}" == "main" ]]; then
            echo "DOCKER_IMAGE_TAG=main" >> $GITHUB_ENV
          else
            echo "DOCKER_IMAGE_TAG=nightly" >> $GITHUB_ENV
          fi
```